src/App.tsx

```diff
@@ -14,11 +14,70 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGood: string,
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1 className="App__title">{this.state.selectedGood.length > 1 ? `${this.state.selectedGood} is selected` : 'No goods selected'}</h1>
+        <button
+          type="button"
+          className={`App__clearButton button is-light ${this.state.selectedGood === '' ? 'App__clearButton--hide is-hidden' : ''}`}
```

Comment:
Will be better to use classnames library
It's too easy to install it in your project and use
https://www.npmjs.com/package/classnames